internal/appsec/listener/grpc/grpc.go

```diff
@@ -0,0 +1,158 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package grpc
+
+import (
+	"sync"
+	"time"
+
+	"go.uber.org/atomic"
+
+	"github.com/DataDog/appsec-internal-go/limiter"
+	waf "github.com/DataDog/go-libddwaf/v2"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/grpcsec"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/sharedsec"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/listener/http"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/listener/shared"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/samplernames"
+)
+
+// gRPC rule addresses currently supported by the WAF
+const (
+	GRPCServerRequestMessage  = "grpc.server.request.message"
+	GRPCServerRequestMetadata = "grpc.server.request.metadata"
+	HTTPClientIPAddr          = http.HTTPClientIPAddr
+	UserIDAddr                = http.UserIDAddr
+)
```

Comment:
They used to be private, and I think they still can be private.

Reply:
They are currently referenced by some tests outside of that package... and it makes it somewhat more convenient/elegant to share addresses.